### PR TITLE
Contact->Communication on report download <PD-1523>

### PR DESCRIPTION
**Pull request by tdphillips**
Fix provided by @aspidites with some modification. Objection?

Tests pass.

### CODE REVIEW

File: myreports/views.py [hunk 1]
```diff
@@ -299,7 +299,17 @@ def downloads(request):
                          if field not in blacklist[report.model]])
 
         values = json.loads(report.values) or fields
+        if 'contact_type' in values:
+            values[values.index('contact_type')] = 'communication_type'
         fields = values + [field for field in fields if field not in values]
+        if 'contact_type' in fields:
```
Comment (aspidites) [inline myreports/views.py]: But you aren't appending. You're in-place editing. Just lines 306 and 312 should be sufficient, no?

Comment (tdphillips) [inline myreports/views.py]: What is now line 304 adds it back.

Comment (tdphillips) [inline myreports/views.py]: If I do this in reverse, however, (construct fields and then filter both), I think I could get by without this if.

Comment (aspidites) [inline myreports/views.py]: Ah, ok. Beware of the column order on the download page (selected values should come before other fields). Couple of other ways to get rid of the inner loop (using a set for instance).

**Comment by tdphillips** [comment]
http://ec2-54-172-15-197.compute-1.amazonaws.com/jenkins/job/MyJobs-Single-Test/85/console

**Pull request merged**